docs/Manuals/Program/freeciv21-server.rst

```diff
@@ -0,0 +1,175 @@
+..  SPDX-License-Identifier: GPL-3.0-or-later
+..  SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+
+freeciv21-server
+****************
+
+SYNOPSIS
+========
+
+``freeciv21-server`` [ -A|--Announce `<PROTO>` ] [ -B|--Bind-meta `<ADDR>` ] [ -b|--bind `<ADDR>` ]
+[ -d|--debug `<LEVEL>` ] [ -e|--exit-on-end ] [ -F|--Fatal ] [ -f|--file `<FILE>` ] [ -i|--identity `<ADDR>` ]
+[ -k|--keep ] [ -l|--log `<FILE>` ] [ -M|--Metaserver `<ADDR>` ] [ -m|--meta ] [ -p|--port `<PORT>` ]
+[ -q|--quitidle `<TIME>` ] [ -R|--Ranklog `<FILE>` ] [ -r|--read `<FILE>` ] [ -S|--Serverid `<ID>` ]
+[ -s|--saves `<DIR>` ] [ -t|--timetrack ] [ -w|--warnings ] [ --ruleset `<RULESET>` ]
+[ --scenarios `<DIR>` ] [ -a|--auth ] [ -D|--Database `<FILE>` ] [ -G|--Guests ] [ -N|--Newusers ]
+[ -h|--help ] [ --help-all ] [ -v|--version ]
+
+
+DESCRIPTION
+===========
+
+Freeciv21 is a free turn-based empire-building strategy game inspired by the history of human civilization.
+The game commences in prehistory and your mission is to lead your tribe from the Stone Age to the Space Age.
+
+Players of Civilization II\ |reg| by Microprose\ |reg| should feel at home. Freeciv21 takes its roots in the
+well-known FOSS game Freeciv and extends it for more fun, with a revived focus on competitive
+:strong:`multiplayer environments`.
+
+Freeciv21 is maintained by the team over at Longturn.net.
+
+An HTML version of this manual page along with much more information is available on our documentation
+website at https://longturn.readthedocs.io/.
+
+This is the server `program` used to establish a Freeciv21 server. For more information on the game, refer
+to freeciv21-client(6).
+
+This manual page only lists the command-line arguments for the server `program`. For details of the
+directives a server administrator can use to control aspects of the server, see freeciv21-game-manual(6).
+
+
+OPTIONS
+=======
+
+The following options are accepted on the command-line of the server. They may not be combined as with other
+tools. For example: ``freeciv21-server -fp savegame.sav 5556`` will not work. Instead you will need to enter
+each option separately, such as, ``freevic21-server -s savegame.sav -p 5556``.
+
+.. include:: /Manuals/Server/command-line.rst
+    :start-line: 15
+
+
+EXAMPLES
+========
+
+Here are some examples:
+
+freeciv21-server --file oldgame.sav -p 2224
+  Starts a server on port `2224`, loading the savegame named `oldgame.sav`.
+
+freeciv21-server -R ranklog -l logfile -r start_script -f oldgame.sav.xz -p 2224
+  Starts a server on port `2224`, loading the savegame named `oldgame.sav.xz`. Player ranking events are
+  logged to `ranking`, other logging events are placed in `logfile`. When the server first starts, it
```

Comment:
Should `ranking` say `ranklog` instead in this sentence? The command is using `ranklog`